sync-minio-server-docs.sh

```diff
@@ -14,6 +17,58 @@ function main() {
 
    mv xx00 source/includes/common-metrics-bucket.md
    mv xx01 source/includes/common-metrics-resource.md
+
+   # Extract metrics v3 list
+
+   # Get the full list
+
+   curl --retry 10 -Ls https://raw.githubusercontent.com/minio/minio/master/docs/metrics/v3.md | csplit - /"## Metric Categories"/
+
+   # Ignore xx00, contains intro text
+   # Overwritten in second csplit anyway
+
+   # Split remainder of file into categories by H3
+   # Number of times to match must be exact
+   cat xx01 | csplit - /"^### "/ '{9}'
+
+   # Copy each category into an include file
+   # Order below must match section order in source file
+
+   # Ignore xx00 again, more intro text
+   rm xx00
+
+   # API (Renamed from Request)
+   # Hopefully ex works the same on both Mac and Linux, unlike sed
+   ex -sc '%s/### Request metrics/### API metrics/' -c 'x' xx01
```

Comment:
What I'm calling API is "Request" in the original GitHub md file. Left it that way Because Reasons, so this is the _fix it in post_ solution.

Reply:
:this-is-fine.gif: